src/azure-cli/azure/cli/command_modules/resource/_win_vt.py

```diff
@@ -47,7 +47,7 @@ def _set_conout_mode(mode):
 def _update_conout_mode(mode):
     old_mode = _get_conout_mode()
     if old_mode & mode != mode:
-        mode = old_mode | mode
+        mode = old_mode | mode  # pylint: disable=unsupported-binary-operation
```

Comment:
Temporarily disable the checker due to `pylint` bug https://github.com/PyCQA/pylint/issues/4526.